account_multicurrency_revaluation/report/currency_unrealized_report.py

```diff
@@ -26,6 +27,13 @@ def __init__(self, account):
     def __contains__(self, key):
         return hasattr(self, key)
 
+    def _format_float(self, val):
+        ndigits = dp.get_precision('Account')(self.cursor)[1]
+        val_formated = val
+        if isinstance(val, float):
+            val_formated = "%.2f" % tools.float_round(val, ndigits)
```

Comment:
You are doing the double rounding here. You round in your format with `%.2f` as well as with `flat_round`. Keep the latter.

Reply:
@mpanarin  the first is for the value the second is for the printing 
even if I don't do :
 `"%.2f"` 1.00 is printed like 1.0

Reply:
I am in this case
```python
  >>> 125650429603636838/(2**53)
  13.949999999999999

  >>> 234042163/(2**24)
  13.949999988079071

  >>> a=13.946
  >>> print(a)
  13.946
  >>> print("%.2f" % a)
  13.95
  >>> round(a,2)
  13.949999999999999
  >>> print("%.2f" % round(a,2))
  13.95
```
https://stackoverflow.com/questions/455612/limiting-floats-to-two-decimal-points

Reply:
https://github.com/odoo/odoo/blob/10.0/odoo/tools/float_utils.py#L136
```py
def float_repr(value, precision_digits):
    """Returns a string representation of a float with the
       the given number of fractional digits. This should not be
       used to perform a rounding operation (this is done via
       :meth:`~.float_round`), but only to produce a suitable
       string representation for a float.
        :param int precision_digits: number of fractional digits to
                                     include in the output
    """
```

Reply:
thanks @simahawk